nrich-search-spring-boot-starter/README.md

```diff
@@ -19,45 +23,41 @@ The artifact is published on [Maven Central Repository](https://search.maven.org
 With Maven:
 
 ```xml
-
 <dependency>
     <groupId>net.croz.nrich</groupId>
     <artifactId>nrich-search-spring-boot-starter</artifactId>
     <version>${nrich.version}</version>
 </dependency>
-
 ```
 
 With Gradle:
 
 ```groovy
-
 implementation "net.croz.nrich:nrich-search-spring-boot-starter:${nrich.version}"
-
 ```
 
 Note if using `nrich-bom` dependency versions should be omitted.
 
 ### Configuration
 
-Configuration is done through a property file, available properties and descriptions are given bellow (all properties are prefixed with nrich.search which is omitted for readability):
+The configuration is done through a property file.
+Available properties and descriptions are given bellow (all properties are prefixed with **nrich.search** which is omitted for readability):
```

Comment:
Not sure about a new line in this case. This sentence is directly related to the previous one.

Reply:
Agreed.